docs/assembly.rst

```diff
@@ -232,9 +232,9 @@ In the grammar, opcodes are represented as pre-defined identifiers.
 +-------------------------+-----+---+-----------------------------------------------------------------+
 | swap1 ... swap16        | `*` | F | swap topmost and ith stack slot below it                        |
 +-------------------------+-----+---+-----------------------------------------------------------------+
-| mload(p)                |     | F | mem[p..(p+32))                                                  |
+| mload(p)                |     | F | mem[p...(p+32))                                                  |
```

Comment:
Thanks for the change! The doc build is failing, and I think the reason is that you have to correctly align the `|` symbols.